TST: PROJ latest failures

https://github.com/pyproj4/pyproj/actions/runs/3920975747/jobs/6702947871
```
FAILED test/test_datum.py::test_datum[Proj] - AssertionError: 
Arrays are not almost equal to 2 decimals

Mismatched elements: 2 / 2 (100%)
Max absolute difference: 7.04815317
Max relative difference: 1.8136988e-06
 x: array([1402291.08, 5076289.59])
 y: array([1402288.54, 5076296.64])
FAILED test/test_datum.py::test_datum[CRS] - AssertionError: 
Arrays are not almost equal to 2 decimals

Mismatched elements: 2 / 2 (100%)
Max absolute difference: 7.04815317
Max relative difference: 1.8136988e-06
 x: array([1402291.08, 5076289.59])
 y: array([1402288.54, 5076296.64])
FAILED test/test_transform.py::test_transform_single_point_nad83_to_nad27 - AssertionError: 
Arrays are not almost equal to 3 decimals

Mismatched elements: 2 / 2 (100%)
Max absolute difference: 14.12664483
Max relative difference: 2.47957478e-05
 x: array([ 569706.333, 4268817.68 ])
 y: array([ 569720.46, 4268813.88])
FAILED test/test_transform.py::test_transform_tuple_nad83_to_nad27 - AssertionError: 
Arrays are not almost equal to 3 decimals

Mismatched elements: 3 / 3 (100%)
Max absolute difference: 21.00634178
Max relative difference: 5.97198411e-05
 x: array([567705.072, 351727.113, 728558.917])
 y: array([567719.249, 351748.119, 728568.053])
FAILED test/test_transformer.py::test_repr__conditional - AssertionError: assert '<Unknown Tra...\n- undefined' == '<Concatenate...47.74, 86.46)'
  - <Concatenated Operation Transformer: pipeline>
  - Description: Inverse of NAD83 to WGS 84 (1) + UTM zone 17N
  + <Unknown Transformer: unknown>
  + Description: unavailable until proj_trans is called
    Area of Use:
  + - undefined
  - - name: North America - onshore and offshore: Canada - Alberta; British Columbia; Manitoba; New Brunswick; Newfoundland and Labrador; Northwest Territories; Nova Scotia; Nunavut; Ontario; Prince Edward Island; Quebec; Saskatchewan; Yukon. United States (USA) - Alabama; Alaska (mainland); Arizona; Arkansas; California; Colorado; Connecticut; Delaware; Florida; Georgia; Idaho; Illinois; Indiana; Iowa; Kansas; Kentucky; Louisiana; Maine; Maryland; Massachusetts; Michigan; Minnesota; Mississippi; Missouri; Montana; Nebraska; Nevada; New Hampshire; New Jersey; New Mexico; New York; North Carolina; North Dakota; Ohio; Oklahoma; Oregon; Pennsylvania; Rhode Island; South Carolina; South Dakota; Tennessee; Texas; Utah; Vermont; Virginia; Washington; West Virginia; Wisconsin; Wyoming.
  - - bounds: (-172.54, 23.81, -47.74, 86.46)
```

--- a/test/test_transformer.py
+++ b/test/test_transformer.py
@@ -565,48 +565,30 @@ def test_repr(from_crs, to_crs, expected_repr):
 
 @pytest.mark.grid
 def test_repr__conditional():
-    trans_repr = repr(Transformer.from_crs(4326, 26917))
-    if grids_available(
-        "ca_nrc_NA83SCRS.tif",
-        "us_noaa_FL.tif",
-        "us_noaa_MD.tif",
-        "us_noaa_TN.tif",
-        "us_noaa_gahpgn.tif",
-        "us_noaa_kyhpgn.tif",
-        "us_noaa_mihpgn.tif",
-        "us_noaa_nchpgn.tif",
-        "us_noaa_nyhpgn.tif",
-        "us_noaa_ohhpgn.tif",
-        "us_noaa_pahpgn.tif",
-        "us_noaa_schpgn.tif",
-        "us_noaa_vahpgn.tif",
-        "us_noaa_wvhpgn.tif",
-    ):
+    trans_repr = repr(Transformer.from_crs("EPSG:4326+3855", "EPSG:4979"))
+    if grids_available("us_nga_egm08_25.tif"):
         assert trans_repr == (
             "<Unknown Transformer: unknown>\n"
             "Description: unavailable until proj_trans is called\n"
             "Area of Use:\n- undefined"
         )
+    elif PROJ_GTE_92:
+        assert trans_repr == (
+            "<Unknown Transformer: noop>\n"
+            "Description: Transformation from EGM2008 height to WGS 84 "
+            "(ballpark vertical transformation, without ellipsoid height "
+            "to vertical height correction)\n"
+            "Area of Use:\n- undefined"
+        )
     else:
         assert trans_repr == (
-            "<Concatenated Operation Transformer: pipeline>\n"
-            "Description: Inverse of NAD83 to WGS 84 (1) + UTM zone 17N\n"
+            "<Other Coordinate Operation Transformer: noop>\n"
+            "Description: Transformation from EGM2008 height to WGS 84 "
+            "(ballpark vertical transformation, without ellipsoid height "
+            "to vertical height correction)\n"
             "Area of Use:\n"
-            "- name: North America - onshore and offshore: Canada - Alberta;"
-            " British Columbia; Manitoba; New Brunswick; "
-            "Newfoundland and Labrador; Northwest Territories; "
-            "Nova Scotia; Nunavut; Ontario; Prince Edward Island; Quebec; "
-            "Saskatchewan; Yukon. United States (USA) - Alabama; "
-            "Alaska (mainland); Arizona; Arkansas; California; Colorado; "
-            "Connecticut; Delaware; Florida; Georgia; Idaho; Illinois; "
-            "Indiana; Iowa; Kansas; Kentucky; Louisiana; Maine; Maryland; "
-            "Massachusetts; Michigan; Minnesota; Mississippi; Missouri; "
-            "Montana; Nebraska; Nevada; New Hampshire; New Jersey; "
-            "New Mexico; New York; North Carolina; North Dakota; Ohio; "
-            "Oklahoma; Oregon; Pennsylvania; Rhode Island; South Carolina; "
-            "South Dakota; Tennessee; Texas; Utah; Vermont; Virginia; "
-            "Washington; West Virginia; Wisconsin; Wyoming.\n"
-            "- bounds: (-172.54, 23.81, -47.74, 86.46)"
+            "- name: World\n"
+            "- bounds: (-180.0, -90.0, 180.0, 90.0)"
         )
 
 

